src/components/committees/committee.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useTranslations } from "next-intl";
+import styles from "@/styles/components/committees/committee.module.scss"
+import Image from 'next/image'
+import Link from 'next/link';
+
+export async function getStaticProps({ locale }) {
+    return {
+      props: {
+        messages: (await import(`@/messages/${locale}.json`)).default
+      }
+    };
+}
+
+
+export default function CommitteeCard({props}) {
+    const t = useTranslations(`CommitteeCards.${props}`)
+    return (
+        <div className={styles.committee_card}>
+            <Link href={t('path')}>
```

Comment:
It would be better if the Link element wrapped the entire committee_card div instead because right now, clicking over the text does not redirect to the committee description, only clicking the image does. If the link is wrapped around the entire div, it'll redirect no matter where the user presses within that maroon box.